frontend/src/strings/crawl-workflows/labels.ts

```diff
@@ -0,0 +1,10 @@
+import { msg } from "@lit/localize";
+
+export const labelFor = {
+  autoscrollBehavior: msg("Autoscroll"),
+  autoclickBehavior: msg("Autoclick"),
+  pageLoadTimeoutSeconds: msg("Page Load Limit"),
+  postLoadDelaySeconds: msg("Delay After Page Load"),
+  behaviorTimeoutSeconds: "Behavior Limit",
+  pageExtraDelaySeconds: msg("Delay After Behaviors"),
```

Comment:
I think we might be better off not renaming this label. The help text and user guide already mention the delay happens after behaviors, and the current label is a little better aligned with the most common use case (pause between pages to avoid rate limiting).